### PR TITLE
Contact me (Code review)

### PR DESCRIPTION
**Pull request by benshidanny11**
- Set the form layout
- Added form title, description, and form controls
- Styled the form

### CODE REVIEW

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @benshidanny11, 

**This is a great project you have going on, well-done lad**. 

   
**However, there are a few sections to improve before your project is approved.** 🔴 
<hr>

## Required Changes ♻️
_Check the inline comments under the review for sections to improve._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: home.html [hunk 1]
```diff
@@ -1,322 +1,420 @@
...
-
-    </main>
-  </div>
-</body>
-
-</html>
\ No newline at end of file
+        </section>
+        <section class="contact-container">
+          <form class="contact-form">
```
Comment (mke2111) [inline home.html]: - [ ] The Form should use the `action` and `method` attributes to integrate with [formspree service](https://formspree.io/html). Ensure that the form is submitted successfully with valid input values

File: home.html [hunk 2]
```diff
@@ -1,322 +1,423 @@
...
+                name="emailaddress"
+                required
+              />
+              <textarea
+                class="input-form textarea"
+                maxlength="500"
+                minlength="10"
+                name="message"
+                placeholder="Write your message here..."
+              ></textarea>
```
Comment (dasileker) [inline home.html]: - [ ] This Also should have a **`required`**, because it will be bad to send an empty message.

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @benshidanny11** 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AshakaE — APPROVED** [review]
### Hi @benshidanny11  🙋‍♀️, 

# STATUS APPROVED ✔
Thanks for making the required changes 😄 
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**